pull_request_template.md

```diff
@@ -1,34 +1,57 @@
 ### 📃 Summary
 <!-- Enter a short description about what this PR contains. -->
-
+---
 ### 🔍 Changeset
 <!-- List out the code changes introduced by this PR. Data Migrations, Endpoints, etc -->
-
+---
 ### 🏷 JIRA Task
 <!-- Provide a link to the associated JIRA task. -->
-
+---
 ### ↩️ Depends On
 <!-- Provide a link to any other PR dependencies. -->
-
+---
 ### 📸 Screenshots (before/after)
 <!-- If this PR alters any UI, provider before/after screenshots. -->
-
+---
 ### 🧪 QA
 <!-- List out the steps to test the feature. -->
 
+- Unit tests
+  - [ ] Yes
+  - [ ] No
```

Comment:
I know for care applications this is enforced by tools like codecov - thinking we may want to remove in favor of avoiding manual inputs for things we can check automatically?

Reply:
@rosaliejames makes sense, we can remove this.